msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2519783.kt

```diff
@@ -0,0 +1,115 @@
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.broker.mwpj
+
+import com.microsoft.identity.client.msal.automationapp.R
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest
+import com.microsoft.identity.client.ui.automation.annotations.LocalBrokerHostDebugUiTest
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost
+import com.microsoft.identity.client.ui.automation.rules.LoadLabUserTestRule
+import com.microsoft.identity.labapi.utilities.client.ILabAccount
+import com.microsoft.identity.labapi.utilities.client.LabQuery
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment
+import com.microsoft.identity.labapi.utilities.constants.TempUserType
+import com.microsoft.identity.labapi.utilities.constants.UserType
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2519783
+@LocalBrokerHostDebugUiTest
+@SupportedBrokers(brokers = [BrokerHost::class])
+class TestCase2519783 : AbstractMsalBrokerTest() {
+
+    private lateinit var mUsGovLabAccount: ILabAccount
+
+    @get:Rule
+    val loadAdditionalLabUserRule: TestRule = LoadLabUserTestRule(getAdditionalLabQuery())
+
+    @Test
+    fun test_2519783() {
+
+        val username = mLabAccount.username
+        val password = mLabAccount.password
+        val usGovUsername = mUsGovLabAccount.username
+        val usGovPassword = mUsGovLabAccount.password
+        val brokerHostApp = broker as BrokerHost
+        brokerHostApp.enableMultipleWpj()
+        brokerHostApp.performDeviceRegistrationMultiple(username, password)
+        brokerHostApp.performDeviceRegistrationMultiple(usGovUsername, usGovPassword)
+        val deviceRegistrationRecords = (broker as BrokerHost).allRecords
+        brokerHostApp.installCertificateMultiple(deviceRegistrationRecords[0]["TenantId"] as String)
+        brokerHostApp.installCertificateMultiple(deviceRegistrationRecords[1]["TenantId"] as String)
```

Comment:
Do we assert something here?

Reply:
nope the method itself verify the installation is successful